web/views.py

```
@@ -6469,6 +6498,61 @@ def all_study_groups(request):
+            f"{request.user.get_full_name() or request.user.username} has self-reported attendance for "
+            f"'{session.title}' on {session.start_time.strftime('%Y-%m-%d')} and is pending your verification."
+        )
+        try:
+            send_mail(
+                subject,
+                message,
+                settings.DEFAULT_FROM_EMAIL,
+                [session.course.teacher.email],
+                fail_silently=True,
+            )
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)